packages/sources/src/components/image/image.scss

```diff
@@ -0,0 +1,22 @@
+.img-responsive {
+  @include img-responsive;
+}
+
+.img-rounded {
+  border-radius: $border-radius-large;
+}
+
+.img-thumbnail {
+  padding: $thumbnail-padding;
+  line-height: $line-height-base;
+  background-color: $thumbnail-bg;
+  border: 1px solid $thumbnail-border;
+  border-radius: $thumbnail-border-radius;
+  transition: all .2s ease-in-out;
```

Comment:
Waar is die transition voor nodig?

Reply:
Meegenomen vanuit `/libs/bootstrap/`, maar ik zie hier ook geen waarde meer voor. Heb hem weggehaald